scripts/zones/Sauromugue_Champaign/mobs/Climbpix_Highrise.lua

```diff
@@ -7,6 +7,10 @@ mixins = {require("scripts/mixins/job_special")}
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 300)
+end
+
```

Comment:
Can you educate me on what is happening here? Something happening when the mob no longer becomes idle? Assumption being after 300s or 300ms, 2hrs become available

Reply:
Just makes the mob despawn if idle for 5 min